docs/catalog-restrictions.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Filtering Broker Catalogs
+layout: docwithnav
+---
+
+# Catalog Restrictions
+
+Services provided by service brokers are represented in Kubernetes by two 
+different [resources](resources.md), service classes and service plans. When a 
+`ClusterServiceBroker` or `ServiceBroker` resource is created, the Service 
+Catalog will query the Service Broker for the list of available Services. 
+Service Catalog will then create `ClusterServiceClass` or `ServiceClass` 
+resources to represent the service classes and `ClusterServicePlan` or 
+`ServicePlan` resources to represent service plans. By default, Service Catalog
+will create a `ClusterServiceClass` or `ServiceClass` for each service class 
+and a `ClusterServicePlan` or `ServicePlan` for each service plan 
+provided by the service broker. When creating a `ClusterServiceBroker` or 
+`ServiceBroker` resource, you can change this behavior by specifying one or 
+more catalog restrictions. Catalog restrictions act in a manner similar to 
+Kubernetes label selectors to enable you to control how service classes and 
+service plans should be exposed from the service brokers.
+
+## Using Catalog Restrictions
+
+Catalog restrictions are specified in `ClusterServiceBroker` or `ServiceBroker`
+ resources. A sample YAML might look like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ClusterServiceBroker
+metadata:
+  name: sample-broker
+spec:
+  authInfo:
+    basic:
+      secretRef:
+        name: sample-broker-auth
+        namespace: brokers
+  catalogRestrictions:
+    servicePlan:
+    - "spec.externalName==basic"
+  url: http://sample-broker.brokers.svc.cluster.local
+```
+
+In this example, a catalog restriction has been defined that specifies that 
+only service plans that have an external name of basic should be selected. 
+Catalog restrictions are defined as a set of one or more rules that target 
+either service classes and service plans. These rules have a special format
```

Comment:
> Catalog restrictions are defined as a set of one or more rules that target either service classes and service plans

Because of the word *either*, I think *and* should be an *or*.  Perhaps we should say "that target service classes and/or service plans"

Reply:
Ahh yeah thanks!  Forgot to remove that word. You can supply both. so and/or